be/src/exprs/split_part.cpp

```diff
@@ -34,9 +122,9 @@ StatusOr<ColumnPtr> StringFunctions::split_part(FunctionContext* context, const
     RETURN_IF_COLUMNS_ONLY_NULL(columns);
 
     if (columns[2]->is_constant()) {
```

Comment:
it seems there exists a SIMD algorithm for split_part when delimiter is constant, the sketch of the algo as follows:
1. search the delimiter BinaryColumn::bytes() and save the offsets of occurrences in a array.
2. then compute the split_port from BinaryColumn' offsets and the offsets from step 1.

We can leave a TODO to notice somebody for future optimization.